openlibrary/templates/book_providers/wikisource_read_button.html

```diff
@@ -0,0 +1,17 @@
+$def with(wikisource_id)
+
+<div class="cta-button-group">
+    <a href="https://wikisource.org/wiki/$wikisource_id" title="$_('Read eBook on Wikisource')"
+        class="cta-btn cta-btn--available cta-btn--read cta-btn--external cta-btn--wikisource" target="_blank"
+        aria-haspopup="true" aria-controls="wikisource-toast">$_('Read')</a>
```

Comment:
This is just an outbound link to Wikisource. I would have liked to include a PDF button as well, but the /pdf/ route that the [Wikimedia API describes](https://en.wikipedia.org/api/rest_v1/#/Page%20content/get_page_pdf__title_) is not available for Wikisource.

Reply:
Another challenge is that this treats "The_Annotated_Strange_Case_of_Dr_Jekyll_and_Mr_Hyde," for example, as an ID, because it's what would need to be used to navigate to the book on Wikisource. But if the title of the work changes, so would the URL, and then our ID wouldn't work anymore.